### PR TITLE
MAT-9365 prevent edit when there is lock

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-9365](https://jira.cms.gov/browse/MAT-9365)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included and Code coverage is verified.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/cqllibraryservice/controllers/CqlLibraryController.java [hunk 1]
```diff
@@ -183,13 +184,19 @@ public ResponseEntity<CqlLibrary> updateCqlLibrary(
...
+              + persistedLibrary.getCqlLibraryLock().getLockedBy());
+    }
     AuthUtils.checkAccessPermissions(persistedLibrary, username);
     if (!persistedLibrary.isDraft()) {
       throw new InvalidResourceStateException("CQL Library", id);
     }
     if (cqlLibraryService.isCqlLibraryNameChanged(cqlLibrary, persistedLibrary)) {
       cqlLibraryService.checkDuplicateCqlLibraryName(cqlLibrary.getCqlLibraryName());
     }
+    // cqlLibraryService.checkCqlLibraryLocked(cqlLibrary.getId(), username);
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/cqllibraryservice/controllers/CqlLibraryController.java]: Dead code?

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/cqllibraryservice/controllers/CqlLibraryController.java]: @jkotanchik-SB yeah, just removed it thanks Joe!

**Comment by adongare** [comment]
@CeciliaLiu8 this may not be required anymore. I've merged something similar https://github.com/MeasureAuthoringTool/cql-library-service/pull/162

**Comment by sb-cecilialiu** [comment]
> @CeciliaLiu8 this may not be required anymore. I've merged something similar #162

Thanks so much Ashok! I'll close this PR.
